mesheryctl/Makefile

```diff
@@ -2,7 +2,7 @@
 GIT_VERSION=$(shell git describe --tags `git rev-list --tags --max-count=1`)
 GIT_COMMITSHA=$(shell git rev-list -1 HEAD)
 RELEASE_CHANNEL="edge"
-LDFLAGS="-w -s -X github.com/meshery/meshery/mesheryctl/internal/cli/root.version=${GIT_VERSION} -X github.com/meshery/meshery/mesheryctl/internal/cli/root.commitsha=${GIT_COMMITSHA} -X github.com/meshery/meshery/mesheryctl/internal/cli/root.releasechannel=${RELEASE_CHANNEL}"
+LDFLAGS="-w -s -X github.com/layer5io/meshery/mesheryctl/internal/cli/root/constants.version=${GIT_VERSION} -X github.com/layer5io/meshery/mesheryctl/internal/cli/root/constants.commitsha=${GIT_COMMITSHA} -X github.com/layer5io/meshery/mesheryctl/internal/cli/root/constants.releasechannel=${RELEASE_CHANNEL}"
```

Comment:
@leecalcote This is the solution for the version thing.

Reply:
Aha! Ok, thank you, @piyushsingariya.

@nehaLohia27, you, and @piyushsingariya are correct. This does fix the other issue. This is something of an urgent fix that we might need to pull out and push separately.